cmd/run.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+var runCmd = &cobra.Command{
+	Use:   "run <image> [command]",
+	Short: "run a command in a new container",
+	Args:  cobra.MinimumNArgs(1),
+	RunE:  run,
+}
+
+func run(_ *cobra.Command, args []string) error {
+	image := args[0]
+	var command []string
+
+	if len(args) >= 2 {
+		command = args[1:]
+	} else {
+		defaultCmd, err := getDefaultCommand(image)
+		if err != nil {
+			return fmt.Errorf("error getting default command: %v", err)
+		}
+		command = defaultCmd
+	}
+
+	fmt.Println("Running", command)
```

Comment:
this feels more like logging

Reply:
IMO a CLI is allowed to print "clean" messages

Reply:
yes, but as user you expect relevant output. If kubectl would return `running get pod` everytime I do something it would be bad ux :smile:

Reply:
for internal output there is debug flag imo

Reply:
> yes, but as user you expect relevant output. If kubectl would return `running get pod` everytime I do something it would be bad ux 😄

You are not required to provide a custom command. If you don't provide one, the default command specified in the image will be used. However, including information about the command being run can be useful in this case.

See: 
```
$ ./container-runtime run ubuntu
Running [/bin/bash]
```

Reply:
and it's nit/not really relevant